pkg/cmd/init.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
```

Comment:
I followed what we seem to be doing in other files which is leaving a space between standard library imports and external ones. I can change it if preferred but `gofmt` will then sort them alphabetically so it will look something like:
```
	"fmt"
	"github.com/ClusterHQ/dvol/pkg/datalayer"
	"github.com/spf13/cobra"
	"io"
	"os"
```

I don't have a strong preference but I'd like for us to be consistent.